### PR TITLE
Product list p0

### PR DESCRIPTION
**Pull request by shraddha15-creator**
Preview : https://deploy-preview-4--upbeat-store.netlify.app/screens/products-list/products.html

*Will make this responsive at the end*
- Added Product Listing page

-- Filters
--  by category
--   by rating

- sort 
-- Products 
-- Showing result of all products

products-listing

### CODE REVIEW

**Review by shubhambishnoi — APPROVED** [review]

File: screens/products-list/products.css [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+.card-product-brand,
+.card-product-name,
+.card-product-price,
+.card-product-offer {
+    font-weight: 500;
+}
+
+/* Add to cart btn  */
+.btn-dark {
+    margin: 0.5rem 0 0 0;
```
Comment (jeetsdev) [inline screens/products-list/products.css:122]: You can use `margin-top: 0.5rem;` here.

**Review by jeetsdev — APPROVED** [review]
Added a suggestion.

File: screens/products-list/products.css (hunk 1 above)
Comment (abhisekgupta001) [reply screens/products-list/products.css:122]: I think, if she just use margin-top, the other 3 sides might get overwritten, explicitly making them 0 as per requirement is fine.

**Review by ayush65 — APPROVED** [review]

**Review by abhisekgupta001 — APPROVED** [review]

File: screens/products-list/products.css (hunk 1 above)
Comment (jeetsdev) [reply screens/products-list/products.css:122]: That's also a valid point.

Comment (shraddha15-creator) [reply screens/products-list/products.css:122]: Thank you @jeetsdev @abhisekgupta001 :)

**Pull request merged**